examples/drivers/windows/otApi/otApi.cpp

```diff
@@ -1347,7 +1348,7 @@ otThreadDiscover(
         aInstance->InterfaceGuid, aCallback, aCallbackContext
         );
 
-    PackedBuffer4<GUID,uint32_t,uint16_t, bool> Buffer(aInstance->InterfaceGuid, aScanChannels, aPanid, aJoiner);
+    PackedBuffer5<GUID,uint32_t,uint16_t, bool, bool> Buffer(aInstance->InterfaceGuid, aScanChannels, aPanid, aJoiner, aEnableEui64Filtering);
```

Comment:
I was just looking at this more closely, and I don't think we should use `bool` here (for either of the last two types), and instead use `uint8_t` to explicitly set the size. `iocontrol.c` would also need to be updated.